### PR TITLE
Global: Add base readme & contribution guide

### PR DESCRIPTION
**Pull request by Fallion**

### CODE REVIEW

File: contributing.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+```
+BLKN: Add React
+```
+
+Please also refer to https://chris.beams.io/posts/git-commit/ for how to write commit messages.
+
+## Commit History
+
+We are using merges in this project without squashing. Please pay attention that your commits are make sense and aren't broken up into illogical chunks. Please squash on your branch if you have any WIP commits.
```
Comment (VaclavSir) [inline contributing.md]: * "your commits make sense" or "your commits are making sense"
* "squash your branch"

Comment (Fallion) [reply contributing.md]: Wow. I'm bad at English today.

File: contributing.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# Contribution guide
+
+## Table of Contents
+
+* [Commit Text](#commit-text)
+* [Commit history](#commit-history)
+
+---
+
+## Commit Text
```
Comment (jakubzaba) [inline contributing.md]: It should be `Commit message`.

File: readme.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+## Repository Structure
+
+This repository is broken up into 3 projects.
+
+| Project | Technology                                               | Description                     |
+| :------ | :------------------------------------------------------- | :------------------------------ |
+| BLKN    | [Styled JSX](https://github.com/zeit/styled-jsx)         | All Components for BLKN project |
+| Kiwicom | [Styled Components](https://www.styled-components.com)   | All Components for Kiwi.com     |
+| Icons   | [React-icons](https://gorangajic.github.io/react-icons/) | All Kiwi Icons                  |
```
Comment (VaclavSir) [inline readme.md]: What? I was supposed to integrate react-icons somehow?

Comment (Fallion) [reply readme.md]: I thought you were using those. Whatever tell me what to put there and I'll update it.

Comment (VaclavSir) [reply readme.md]: No, the Frontend team is using react-icons on the Frontend project, but I thought I should add our custom icons. Now I'm even less sure what is the expected output of my current task...

Comment (Fallion) [reply readme.md]: Yes... the task was to add our custom icons... So just tell me what I should put as the technologies used. I don't know what react-icons does and there was a discussion in person that mentioned this. Hence the assumption.

File: contributing.md (hunk 2 above)
Comment (Fallion) [reply contributing.md]: Updated

File: readme.md (hunk 3 above)
Comment (oreqizer) [reply readme.md]: `react-icons` takes SVG versions of material icons, font awesome etc. and makes them into components

**Pull request merged**